graphstorm-processing/graphstorm_processing/data_transformations/dist_transformations/dist_label_transformation.py

```diff
@@ -75,10 +76,18 @@ def apply(self, input_df: DataFrame) -> DataFrame:
             *original_cols,
         )
 
+        # Create schema for label_df
```

Comment:
What if the label column name is not like the format "idx:Int"?

Reply:
The `idx` column is one we create through the call to `list(enumerate(str_indexer_model.labelsArray[0]))`